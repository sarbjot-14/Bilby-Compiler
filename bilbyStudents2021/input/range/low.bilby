main{
	
     imm result := low <4 .. 7> ;
    print result $n;
    
    imm resultFloat := low <4.4 .. 7.4> ;
    
    print resultFloat $n;
    
    imm resultChar := low <#a .. #f> ;
    
    print resultChar $n;
	
}