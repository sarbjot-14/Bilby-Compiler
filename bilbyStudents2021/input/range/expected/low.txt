4
4.400000
a
